.travis/script.sh

```diff
@@ -626,6 +626,12 @@ build_samr21() {
     REFERENCE_DEVICE=1 COVERAGE=1 PYTHONUNBUFFERED=1 OT_NCP_PATH="$(pwd)/$(ls output/posix/*/bin/ot-ncp)" RADIO_DEVICE="$(pwd)/$(ls output/*/bin/ot-rcp)" NODE_TYPE=ncp-sim make -f src/posix/Makefile-posix check || die
 }
 
+[ $BUILD_TARGET != posix-app-spi ] || {
+    ./bootstrap || die
+    # enable code coverage for OpenThread posix radio
```

Comment:
nit: given that we are not including `COVERAGE=1` here but rather in `.travis.yml`, should we remove this comment here?

Reply:
Already removed the useless comment.